datadog_checks_dev/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG - Datadog Checks Dev
 
+## 15.3.2 / 2021-11-22
```

Comment:
Does the `Added` change bump the version to `15.4.0`?

Reply:
yes let me fix